SlicerDevelopmentToolboxUtils/helpers.py

```diff
@@ -658,7 +658,11 @@ def value(self):
 
   @value.setter
   def value(self, value):
-    self.originalValue = str(value) if value else ""
+    if not value:
+      value = ""
+    if type(value) is not str:
+      value = str(value)
+    self.originalValue = value
```

Comment:
Here's an example how this will fail:

```
In [5]: s=u'йцук'
In [7]: type(s)
Out[7]: unicode
In [8]: str(s)
---------------------------------------------------------------------------
UnicodeEncodeError                        Traceback (most recent call last)
<ipython-input-8-d22ffcdd2ee9> in <module>()
----> 1 str(s)

UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-3: ordinal not in range(128)
```